spring-boot-project/spring-boot-devtools/src/main/java/org/springframework/boot/devtools/restart/RestartApplicationListener.java

```diff
@@ -35,7 +38,7 @@
 public class RestartApplicationListener
 		implements ApplicationListener<ApplicationEvent>, Ordered {
 
-	private int order = HIGHEST_PRECEDENCE;
+	private static final Log logger = LogFactory.getLog(RestartApplicationListener.class);
```

Comment:
Please refrain from doing unrelated changes. This change to `order`  is unrelated to the task at hand.